File: backend/schemas/inbox.py
"""Inbox notification cards — PHASE_08."""

from pydantic import BaseModel, ConfigDict


class InboxNotificationItem(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: str
    report_id: str
    building_name: str
    building_address: str
    score_at_trigger: float
    threshold: int
    genome_archetype: str
    contact_name: str | None
    contact_title: str | None
    contact_company: str | None
    run_label: str
    run_at: str | None
    read_at: str | None

File: frontend/types/inbox.ts
export type InboxNotificationItem = {
  id: string;
  report_id: string;
  building_name: string;
  building_address: string;
  score_at_trigger: number;
  threshold: number;
  genome_archetype: string;
  contact_name: string | null;
  contact_title: string | null;
  contact_company: string | null;
  run_label: string;
  run_at: string | null;
  read_at: string | null;
};
